### PR TITLE
Changes for SDXL - WIP

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: script/install-pytorch-from-src/_cm.json [hunk 1]
```diff
@@ -377,6 +377,31 @@
         }
       ]
     },
+    "for-nvidia-mlperf-inference-v4.1": {
```
Comment (arjunsuresh) [inline script/install-pytorch-from-src/_cm.json:380]: This is not needed right?

File: script/app-mlperf-inference/_cm.yaml [hunk 2]
```diff
@@ -1536,14 +1549,15 @@ docker:
...
   skip_run_cmd: 'no'
   shm_size: '32gb'
   interactive: True
   extra_run_args: ' --ulimit memlock=-1 --cap-add SYS_ADMIN --cap-add SYS_TIME --security-opt apparmor=unconfined --security-opt seccomp=unconfined'
   os: ubuntu
   cm_repo: mlcommons@cm4mlops
-  cm_repo_flags: ' --branch=mlperf-inference '
-  real_run: False
+  cm_repo_flags: ' --branch=mlperf-inference'
+  real_ruf: False
```
Comment (arjunsuresh) [inline script/app-mlperf-inference/_cm.yaml:1560]: typo?

Comment (anandhu-eng) [reply script/app-mlperf-inference/_cm.yaml]: fixed in https://github.com/GATEOverflow/cm4mlops/pull/111/commits/6b3175ba84c57e0f5ba64996222b2480e76e1b9b

File: script/install-pytorch-from-src/_cm.json (hunk 1 above)
Comment (anandhu-eng) [reply script/install-pytorch-from-src/_cm.json]: removed in https://github.com/GATEOverflow/cm4mlops/pull/111/commits/9cd9bf5bb3eac4dab5671a2f94f8d32fdb6e7719

**Review by arjunsuresh — APPROVED** [review]

**Pull request merged**